components/Settings/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, IconButton, Menu, MenuItem, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import SettingsRoundedIcon from '@mui/icons-material/SettingsRounded';
+import AdvancedViewSwitch from 'components/AdvancedSwitch';
+import SwitchTheme from 'components/SwitchTheme';
+import SelectLanguage from 'components/SelectLanguage';
+
+const Settings = () => {
+  const { t } = useTranslation();
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => setAnchorEl(event.currentTarget);
+  const handleClose = () => setAnchorEl(null);
+
+  return (
+    <>
+      <IconButton
+        id="view-contracts-button"
+        aria-controls={open ? 'basic-menu' : undefined}
+        aria-haspopup="true"
+        aria-expanded={open ? 'true' : undefined}
+        onClick={handleClick}
+        size="small"
+        sx={{ outline: '1px solid', width: '32px', height: '32px', borderRadius: '50%', p: '0px' }}
+        // variant="outlined"
+      >
+        <SettingsRoundedIcon sx={{ fontSize: 20 }} />
+      </IconButton>
+      <Menu
+        id="basic-menu"
```

Comment:
this id is duplicated, consider using another one. Remember to also update the `aria-control` attr above with the same